cypress/integration/prospects/note.js

```diff
@@ -0,0 +1,38 @@
+describe('Create a prospect note', () => {
+  it('Should be able to create a prospect note', () => {
+    cy.login('normal@example.com', 'password')
+    cy.visit('http://localhost:3000/prospects/list')
+
+    // open the slide pane to create a prospect
+    cy.get('button[id="create-prospect"').click()
+
+    // fill the text fields for email, name, and phone
+    cy.get('input[id="mail"]').click().type('note@wellcode.com')
+    cy.get('input[id="name"]').click().type('prospect note')
+    cy.get('input[id="phone"]').click().type('0747070781')
+
+    // create a new prospect and check for success message
+    cy.get('button[id="create"').click()
+    cy.contains('Prospect successfully created!')
+
+    // check the existence of the "Save" and "Cancel" buttons
+    cy.get('#save').should('not.exist')
+    cy.get('#cancel').should('not.exist')
+
+    // open the textarea to add a prospect note
```

Comment:
we should split this "big" test into multiple smaller ones. What do you think it's the best way for doing this?

Reply:
Would it be better to divide this test into two tests?

1. **prospect creation** (I think that since this test has already been implemented, there is no point in implementing it again. I am creating this here so that this functionality can be tested individually)

2. **testing the 'CreateProspectNote' functionality**

Reply:
I would split into 2: clicking the create note button and check that the textarea is there and another one for cancel